core/domain/fs_domain.py

```diff
@@ -138,7 +139,7 @@ def _get_gcs_file_url(self, filepath):
         gcs_file_url = '%s/%s' % (self._assets_path, filepath)
         return gcs_file_url
 
-    def isfile(self, filepath):
+    def isfile(self, filepath: str) -> Any:
```

Comment:
Why `Any`?

Reply:
because these return Any when I run the command

Reply:
Not sure what that means, what command?